nf_core/lint.py

```diff
@@ -790,32 +815,27 @@ def check_actions_awstest(self):
             with open(fn, "r") as fh:
                 wf = yaml.safe_load(fh)
 
-            # Check that the action is only turned on for push
+            # Check that the action is only turned on for workflow_dispatch
             try:
-                assert "push" in wf[True]
+                assert "push" not in wf[True]
                 assert "pull_request" not in wf[True]
+                assert "workflow_dispatch" in wf[True]
```

Comment:
```suggestion
                assert "workflow_dispatch" in wf[True]
                assert "push" not in wf[True]
                assert "pull_request" not in wf[True]
```

Reply:
hmm I thought that this was already re-ordered in my PR

Reply:
Yes, I thought so too? 🤔

Reply:
Must have been a different place I think - here it's just one `in`, in the discussion before I remember that there were 2.